src/Core/Grid/Definition/Factory/AttributeGridDefinitionFactory.php

```diff
@@ -0,0 +1,292 @@
+use PrestaShop\PrestaShop\Core\Grid\Action\Row\RowActionCollection;
+use PrestaShop\PrestaShop\Core\Grid\Action\Row\Type\LinkRowAction;
+use PrestaShop\PrestaShop\Core\Grid\Action\Row\Type\SubmitRowAction;
+use PrestaShop\PrestaShop\Core\Grid\Action\Type\LinkGridAction;
+use PrestaShop\PrestaShop\Core\Grid\Action\Type\SimpleGridAction;
+use PrestaShop\PrestaShop\Core\Grid\Column\ColumnCollection;
+use PrestaShop\PrestaShop\Core\Grid\Column\Type\Attribute\ColorColumn;
+use PrestaShop\PrestaShop\Core\Grid\Column\Type\Common\ActionColumn;
+use PrestaShop\PrestaShop\Core\Grid\Column\Type\Common\BulkActionColumn;
+use PrestaShop\PrestaShop\Core\Grid\Column\Type\DataColumn;
+use PrestaShop\PrestaShop\Core\Grid\Filter\Filter;
+use PrestaShop\PrestaShop\Core\Grid\Filter\FilterCollection;
+use PrestaShop\PrestaShop\Core\Hook\HookDispatcherInterface;
+use PrestaShopBundle\Form\Admin\Type\SearchAndResetType;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+
+/**
+ * Defines grid for attributes group > attributes list
+ */
+final class AttributeGridDefinitionFactory extends AbstractGridDefinitionFactory
+{
+    const GRID_ID = 'attribute';
+
+    /**
+     * @var int
+     */
+    private $attributeGroupId;
+
+    /**
+     * @var AttributeGroupViewDataProvider
+     */
+    private $attributeGroupViewDataProvider;
+
+    /**
+     * @param HookDispatcherInterface $hookDispatcher
+     * @param int $attributeGroupId
+     * @param AttributeGroupViewDataProvider $attributeGroupViewDataProvider
+     */
+    public function __construct(
+        HookDispatcherInterface $hookDispatcher,
+        $attributeGroupId,
+        $attributeGroupViewDataProvider
+    ) {
+        parent::__construct($hookDispatcher);
+        $this->attributeGroupId = $attributeGroupId;
+        $this->attributeGroupViewDataProvider = $attributeGroupViewDataProvider;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function getId()
+    {
+        return self::GRID_ID;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function getName()
+    {
+        return $this->attributeGroupViewDataProvider->getAttributeGroupNameById((int) $this->attributeGroupId);
+    }
+
+    /**
+     * {@inheritdoc}
+     *
+     * @throws AttributeGroupException
+     * @throws AttributeGroupNotFoundException
+     */
+    protected function getColumns()
+    {
+        $columns = (new ColumnCollection())
+            ->add((new BulkActionColumn('bulk'))
+                ->setOptions([
+                    'bulk_field' => 'id_attribute',
+                ])
+            )
+            ->add((new DataColumn('id_attribute'))
+                ->setName($this->trans('ID', [], 'Admin.Global'))
+                ->setOptions([
+                    'field' => 'id_attribute',
+                ])
+            )
+            ->add((new DataColumn('value'))
+                ->setName($this->trans('Value', [], 'Admin.Global'))
```

Comment:
Isn't it supposed to be _Values_ instead?

Reply:
According to legacy, here in `Attribute list`(the child list of attribute group) it's supposed to be `Value`, but in `Attribute group list` (the parent list of attribute) it's `Values`

Reply:
My bad, thank you @zuk3975!